ckan/templates/package/snippets/resources.html

```diff
@@ -13,22 +13,22 @@
 #}
 {% set resources = pkg.resources or [] %}
 {% if resources %}
-    {% block resources %}
-	<section class="module module-narrow resources">
-	    {% block resources_inner %}
-		{% block resources_title %}
-		    <h2 class="module-heading"><i class="fa fa-files-o"></i> {{ _("Resources") }}</h2>
-		{% endblock %}
-		{% block resources_list %}
-		    <ul class="list-unstyled nav nav-simple">
-			{% for resource in resources %}
-			    <li class="nav-item{{ ' active' if active == resource.id }}">
-				{% link_for h.resource_display_name(resource)|truncate(25), named_route='resource.%s' % (action or 'read'), id=pkg.name, resource_id=resource.id, inner_span=true %}
-			    </li>
-			{% endfor %}
-		    </ul>
-		{% endblock %}
-	    {% endblock %}
-	</section>
-    {% endblock %}
+	{% block resources %}
```

Comment:
tab?

Reply:
nice spot! dunno how that slipped through!